Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -2323,3 +2325,26 @@ linkRCWithDriverForFleet merchantShortId opCity fleetOwnerId req = do
     driverRCAssoc <- makeRCAssociation driver.merchantId driver.merchantOperatingCityId driver.id rc.id (convertTextToUTC (Just "2099-12-12"))
     QRCAssociation.create driverRCAssoc
   return Success
+
+postDriverClearFee :: (ShortId DM.Merchant -> Context.City -> Maybe Text -> API.Types.ProviderPlatform.Management.Driver.ClearDriverFeeReq -> Flow APISuccess)
+postDriverClearFee _merchantShortId _opCity a1 req = do
+  merchant <- findMerchantByShortId _merchantShortId
+  merchantOpCityId <- CQMOC.getMerchantOpCityId Nothing merchant (Just _opCity)
+  when (isNothing a1) $ throwError $ InvalidRequest "driverId not passed"
```

Comment:
you can use `mandatoryQuery` in spec, then no need to throw error

Reply:
![image](https://github.com/user-attachments/assets/6dc68775-60f0-4af7-85d0-2f9d1b146f6d)
this error is coming on using `mandatoryQuery` 
spec :-
![image](https://github.com/user-attachments/assets/7b2af098-4c12-4a91-9c0c-607f625b29a3)